.github/workflows/test-before-integration.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build test
+
+on:
+  pull_request:
+    branches: [ development ]
```

Comment:
Only pull requests meant to be merged into **development branch** would be tested!